docusaurus/docs/React/release-guides/new-reactions.mdx

```diff
@@ -0,0 +1,135 @@
+    // you'd need to look for supported id's in stream-emoji.json under "emojis" key
+    id: 'bulb',
+  },
+  // unsupported
+  {
+    id: 'rick_roll',
+  },
+  ...defaultMinimalEmojis,
+];
+```
+
+## Default setup and how it works
+
+SDK by default comes with six reaction types (`angry`, `haha`, `like`, `love`, `sad` and `wow`) which are rendered by [`StreamEmoji`]() component which utilises sprite sheet system and renders images for each reaction to make sure it works flawlessly on every system. Default reaction options are defined in [`defaultReactionOptions`]() and these options are reused for both [`ReactionSelector`]() and [`ReactionList`]() (as well as [`SimpleReactionList`]()). These options come by default from the ComponentContext but local component property will be prioritised if defined:
+
+```ts
+contextReactionOptions = defaultReactionOptions;
+// ...
+const reactionOptions = propsReactionOptions ?? contextReactionOptions;
+```
+
+## Custom reaction types and components
+
+It's possible to supply your own reaction types and components to represent such reactions - let's implement reaction of type `rick_roll` to render a Rick Roll GIF:
+
+```tsx
+// mention custom types and components here
+
+const CustomReactionSelector = forwardRef((props, ref) => (
+  <ReactionSelector {...props} ref={ref} reactionOptions={selectorReactions} />
+));
+
+const CustomReactionsList = (props) => <ReactionsList {...props} reactionOptions={listReactions} />;
+```
+
+## EmojiMart integration
+
+If you're used to work with EmojiMart then integrating with new reaction system shouldn't be a big trouble as you can define how your components look and reach for context if you need to - after all, it's just components.
+
+```tsx
+const CustomLikeComponent = () => {
+  // arbitrary EmojiMartContext (does not come with the SDK)
+  const { selectedSkinTones, selectedSet } = useMyEmojiMartContext();
+
+  return <em-emoji id='+1' set={selectedSet} skin={selectedSkinTones['+1']} />;
+};
+
+const reactionOptions = [
+  {
+    type: 'like',
+    component: CustomLikeComponent,
+    name: 'EmojiMart like',
+  },
+];
+
+// pass reaction options to either ReactionSelector or ReactionList or component context (Channel)
+```
+
+## Use of different components for the same reaction types for ReactionSelector and ReactionList components
+
+Either for small changes like this:
+
+```tsx
+const selectorReactionOptions = [
+  {
+    type: 'like',
+    component: ReactionComponent.Like,
+    name: 'Like',
+  },
+];
+
+const listReactionOptions = [
+  {
+    type: 'like',
+    // in this example it's just different size of the emoji, but can be anything you want it to be
+    component: ReactionComponent.LikeMedium,
+    name: 'Like medium',
+  },
+];
+```
+
+Or completely adjusted sets:
+
+```tsx
+// subset of clickable options available to the user
+const selectedReactionOptions = [{ type: 'haha' /* ... */ }, { type: 'cat' /* ... */ }];
```

Comment:
What `/*...*/` stand for?

Reply:
Other required properties but not essential for this part of the guide, I guess this requires you to read through the beginning because if you were to jump straight to this part it wouldn't make much sense - I'll add other properties as well.